Improved topcmd function

TomfromBerlin opened the pull request:
Added an option to the topcmd function ("-v"). This makes it possible to view not only the history of the commands, but also the associated arguments, or pipes, etc.

TomfromBerlin committed:
Improve function topcmd

The topcmd function now displays table headers and shows the commands including arguments, if any were passed.

TomfromBerlin committed:
Merge pull request #1 from TomfromBerlin/TomfromBerlin-modify_func_topcmd

Improve function topcmd

TomfromBerlin committed:
Enhance topcmd function

Added an option to the topcmd function ("-v"). This makes it possible to view not only the history of the commands, but also the associated arguments or piped output.

abhinav-nath commented:
This doesn't seem to be working on MacOS

<img width="646" alt="image" src="https://github.com/abhinav-nath/all-about-shell/assets/48696735/80bee0e4-edda-4490-b5ec-df959749840a">

<img width="881" alt="image" src="https://github.com/abhinav-nath/all-about-shell/assets/48696735/993f1dc1-e748-41f5-9494-47a64d96fdd1">

TomfromBerlin commented:
That's a pity. On Linux "man column" gives this:

-N, --table-columns names
           specifies the column names in a comma-separated list. These names are used for the table headings or to address the column in option arguments.

Obviously, Apple is once again cooking its own soup here.

It should look like this:

![Screenshot_topcmd-v](https://github.com/abhinav-nath/all-about-shell/assets/123265893/433201a0-b2f0-45c5-bab0-c1b2209af46f)

I can't look for a solution, since I do not run Apple devices. Perhaps this can not be merged.

TomfromBerlin committed:
Update function topcmd

Now the function topcmd will determine which operating system is present based on the existence of `uname` and call the code accordingly.

TomfromBerlin commented:
I close this, since it doesn't work as expected.
